src/source/authentication.rst

```diff
@@ -40,6 +40,11 @@ OAuth 2.0 defines the following roles of users and applications:
   In this context *authorization server* is the server hosting authentication
   and token endpoints.
 
+In our Oauth2 flow we rely on the *tenant* as being the **Authentication Server**, the
+one that confirms hte user identity through the use of some kind of credentials (usually username/password).
+The authentication server represents a user or a service account to the **Authorization Server**.
```

Comment:
what does 'represents' mean in this context? I don't understand this sentence.

Reply:
presents?owns?